src/aditiGJUN20/AllDifferIndexInArray.java

```diff
@@ -0,0 +1,33 @@
+/*
+Program 3: Find all differ index from given arrays. 
+	i/p:- arr1: {10,2,9,14,3}
+    arr2: {10,2,18,14,13}
+    o/p:- Values are not matching at index -> 2
+    Values are not matching at index -> 4
+*/        
+
+package aditiGJUN20;
+
+public class AllDifferIndexInArray {
+	
+	// Method to find array index which has different element value 
+		void getAllDifferIndex(int[] arr1, int[] arr2) {
```

Comment:
Aditi when we start method name as get it means that we are going to return something from that method. 
So please don't use that. Instead of it, you can use findAllDifferIndex() as method name.

Reply:
Method name changed as suggested.